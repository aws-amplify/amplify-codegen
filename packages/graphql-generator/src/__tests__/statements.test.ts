import { generateStatements, GenerateStatementsOptions } from '..';

describe('generateStatements', () => {
  test('basic test', () => {
    const options: GenerateStatementsOptions = {
      schema: `
        type Query {
          hello: String!
        }
        
        schema {
          query: Query
        }
      `,
      appSyncApi: '',
      target: 'javascript',
    };

    const statements = generateStatements(options);
    console.log(statements);
    expect(statements).toMatchSnapshot();
  });
});
