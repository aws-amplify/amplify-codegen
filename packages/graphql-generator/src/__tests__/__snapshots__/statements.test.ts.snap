// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generateStatements basic test 1`] = `
Object {
  "queries.js": "/* eslint-disable */
// this is an auto generated file. This will be overwritten

export const hello = /* GraphQL */ \`
  query Hello {
    hello
  }
\`;
",
}
`;
